projects/starbeat/starbeat.mdx

```diff
@@ -0,0 +1,39 @@
+---
+title: Starbeat
+description: Check out how we helped create the interface that powers Starbeat - an entertainment space focused on stars and fans. Get the best React Native developers to build your product!
+meta:
+- react native solution provider
+- react native services
+keywords:
+- starbeat
+- react native development company
+- react native development company in san francisco
+- react native development companies in bangalore
+- react native dev shop in USA
+link: /starbeat
+---
+
+![Starbeat Screenshot](./images/Screenshot-2019-07-16-18-19-32.png)
+
+The app was in the entertainment space, focused on stars and fans and all the little interactions surrounding them. The client wanted a mobile application(in React Native) that should be smooth and easy to use. It was a complicated application in terms of UI and navigations and the feature list was long.
```

Comment:
Can we change the first line to `The app is focused on stars, their fans and all the little interactions surrounding them.`